### ISSUE TITLE
L'offre zen pour l'été est intégrée sur le site

### ISSUE THREAD

**Issue by annabellou** [Kozea/www-teepy#55]
Suite à https://github.com/Kozea/teepy/issues/2016, voici les éléments à intégrer sur le site institutionnel : 

**Bandeau en haut de la navigation**

- [ ] créer la section html sur toutes les pages
- [ ] ajouter le style css

**Section offre sur la page d'accueil**

- [ ] créer la section avant celle du header actuel
- [ ] insérer le contenu texte
- [ ] ajouter le style css

**Page contact**

- [ ] remonter le formulaire de contact en haut de page
- [ ] ajouter une section en offre zen
- [ ] style la section offre zen
- [ ] modifier le style de la section "profitez d'un diagnostic…"
- [ ] modifier le style de la section "nos services"
- [ ] modifier le style des coordonnées
- [ ] ajouter un nouvel objet de contact dans le select de l'objet de la demande

**Comment by rom-rey** [comment]
Pour la mention sur le logiciel Kabeo, il faut rajouter une explication dans l'onglet "gestion en interne" comme tu as dit tout à l'heure du coup non ?

**Comment by annabellou** [comment]
Exact, je me rajoute un point dans la checklist, merci pour le rappel !
